worker.go

```diff
@@ -181,7 +181,5 @@ func (w *Worker) Flush() []DDMetric {
 //
 // Note that the worker will only stop *after* it has finished its work.
 func (w *Worker) Stop() {
-	go func() {
-		w.QuitChan <- true
-	}()
+	close(w.QuitChan)
```

Comment:
So I left this code — the `QuitChan` stuff in, even though I'm not using it. I'm also happy to take it out for now…